### ISSUE TITLE
"Interval" doesn't clear properly and makes it so you can't save funnel queries.

### ISSUE THREAD

**Issue by wetzler** [keen/explorer#121]
At least 2-3 times today I've had a interval of "minutely" mysteriously appear in the explorer. One time it was when updating a saved query, it changed from monthly to minutely.

Another time, it was after I clicked "new query" that the new query started with a default interval of "minutely".

Another time, it appeared after I clicked "clear". 

Sorry I don't have more details yet!

**Comment by wetzler** [comment]
Ok I'm pretty sure if you CLEAR any query with an interval, it reset it's interval to "minutely".

**Comment by dustinlarimer** [comment]
Adding a bit more info here: the Interval component will always have a value, but should be toggled "off" when the query builder is cleared out.

**Comment by wetzler** [comment]
This came up again when building a demo for a customer. I created a funnel and then tried to save it. I got an error about "interval" being set.

Steps to repro:
1. Create project and load sample table
2. Run a count of login and signup events, previous 7 days, daily
3. Build a funnel with step 1 signup, steps 2-5 login
4. Click save funnel. 
5. Get an error about interval being set. 

![screen shot 2016-06-07 at 3 31 13 pm](https://cloud.githubusercontent.com/assets/1840448/15876921/51c7f26a-2cc5-11e6-918b-a6c5150c66a0.png)
